# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/shader-slang/hotfix-install.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

diff -ur slang/tools/CMakeLists.txt slang-new/tools/CMakeLists.txt
--- slang/tools/CMakeLists.txt	2000-01-01 01:00:00.000000000 +0100
+++ slang-new/tools/CMakeLists.txt	2026-10-06 14:59:14.501602882 +0200
@@ -219,6 +219,7 @@
     set(modules_dest_dir $<TARGET_FILE_DIR:slangc>)
     add_custom_target(
         copy-gfx-slang-modules
+        ALL
         COMMAND ${CMAKE_COMMAND} -E make_directory ${modules_dest_dir}
         COMMAND
             ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_LIST_DIR}/gfx/gfx.slang
